node/core/pvf/common/src/worker/mod.rs

```diff
@@ -128,6 +128,16 @@ pub fn worker_event_loop<F, Fut>(
 		}
 	}
 
+	// Delete all env vars to prevent malicious code from accessing them.
+	for (key, _) in std::env::vars() {
```

Comment:
```suggestion
	for (key, _) in std::env::vars_os() {
```

It should be safer to use `vars_os` iterator as `vars` may panic on invalid UTF8.